arches/app/datatypes/base.py

```diff
@@ -532,3 +532,9 @@ def validate_node(self, node):
         a GraphValidationError
         """
         pass
+
+    def transform_value_from_tile(self, value, **kwargs):
+        """
+        Transform db value to simpler representation for use on the front-end
+        """
+        return value
```

Comment:
can you put this method underneath the `transform_value_for_tile` method as they are essentially matched pairs.